documentation/commands/git-prune-branches.md

```diff
@@ -12,4 +12,6 @@ git prune-branches
 
 #### DESCRIPTION
 
-Deletes merged branches in the local and remote repositories.
+Deletes branches whose remote branch no longer exists,
+i.e. branches that were shipped or killed on another machine,
+from the local repository.
```

Comment:
Thoughts on rearranging this slightly so we don't have the clarification in the middle of the sentence.
```
Deletes branches whose remote branch no longer exists from the local repository.
(i.e. branches that were shipped or killed on another machine)
```